r/R/compute.R

```diff
@@ -186,6 +186,32 @@ unique.ArrowDatum <- function(x, incomparables = FALSE, ...) {
   call_function("unique", x)
 }
 
+#' @export
+any.ArrowDatum <- function(..., na.rm = FALSE){
+  
+  a <- collect_arrays_from_dots(list(...))
+  result <- call_function("any", a)
+
+  if(!as.vector(result) && a$null_count > 0 && !na.rm){
```

Comment:
Small question: what's the `!as.vector(result)` for?

Reply:
We discussed this just now on a call, but for transparency, I'll answer here too!  `any()` and `all()` use three-valuered logic, so here if there are `NA`s in the data and `na.rm` is set to `FALSE`, we return an NA if the initial result is not `TRUE`.